_data/projects/jmeta.yml

```diff
@@ -0,0 +1,12 @@
+name: JMeta
+desc: JMeta is a free software project, written in Java, aiming to provide a search platform and a framework for building fully-decentralized P2P search applications.
+site: https://gitlab.ouahpiti.info/meta/jmeta
+tags:
+- p2p
+- "search engine"
+- DHT
+- metadata
+- up-for-grabs
+upforgrabs:
+  name: JMeta issues
```

Comment:
Thank you for the feedback. We will have to fix this quickly as our issues management takes place there.
I've put our github page instead.